shared/common-adapters/oriented-image.desktop.js

```diff
@@ -86,20 +141,14 @@ class OrientedImage extends React.Component<Props, State> {
       return this.setState({styleTransform: _cacheStyleTransforms[src]})
     }
 
-    // EXIF will make an HTTP request locally to 127.0.0.1:* to fetch the
-    // image that the keybase service is serving
-    // img = this refers to the image ArrayBuffer fetched from the local server.
-    const handleData = this._handleData(src)
-    const _hasComponentMounted = () => this._hasComponentMounted
-
-    try {
-      EXIF.getData({src}, function() {
-        if (!_hasComponentMounted()) return
-        handleData(this)
-      })
-    } catch (_) {
-      // Mark src as null in the cache to avoid making subsequent calls.
-      _cacheStyleTransforms[src] = NO_TRANSFORM
+    if (this.props.preview) {
```

Comment:
Since `_readExifLocal` is always for local files and `_fetchExifUploaded` is always for 'remote' files, can this component figure out which to do based on `src` rather than a prop?